neptune/alpha/exceptions.py

```diff
@@ -340,3 +340,24 @@ def __init__(self):
 {correct}Need help?{end}-> TODO: create and link docs
 """.format(**STYLES)
         super().__init__(message)
+
+
+class NeptuneIntegrationNotInstalledException(NeptuneException):
+    def __init__(self, framwork):
+        message = """
```

Comment:
Perhaps make this local variable into a top-level variable? Multi-line strings look ugly in the middle of the method

Reply:
This is done in the exact same way as the rest of this file. Making it top-level doesn't make sense